Black-Market/Black-Market/Components/MarketButton.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  MarketButton.swift
+//  Black-Market
+//
+//  Created by Junior Sancho on 7/10/23.
+//
+
+import SwiftUI
+
+struct MarketButton: View {
+    @Binding var style: Style
+    private let action: () -> Void
+    private let title: String
+        
+    enum Style {
+        case filled
+        case disabled
+        case plain
+    }
+    
+    init(
+        style: Binding<Style> = .constant(.plain),
+        action: @escaping () -> Void,
+        title: String
+    ) {
+        self._style = style
+        self.action = action
+        self.title = title
+    }
+
+    var body: some View {
+        Button {
+            action()
+        } label: {
+            Text(title)
+                .applyButtonStyle(style)
+        }
+        .disabled(style == .disabled)
+    }
+}
+
+struct FilledButtonStyle: ViewModifier {
+    
+    func body(content: Content) -> some View {
+        content
+            .frame(maxWidth: .infinity, maxHeight: 44)
```

Comment:
please, move aa constant  in a single file for reusable components and use extensions for specific views. This can help to centralize theses dimensions and easily change in the future in only one place.

Reply:
Done